src/index.js

```diff
@@ -0,0 +1,324 @@
+const OTPAuth = require('otpauth');
+import CryptoJS from "crypto-js";
+import browser from 'browser-detect';
+import crypto from 'crypto';
+import eccrypto from 'eccrypto';
+import BeetClientDB from './lib/BeetClientDB';
+import "isomorphic-fetch";
+
+
+
+
+
+
+
+
+
+class Beet {
```

Comment:
Please add documentation to the head, at least what all the possible flags mean

Reply:
ok. will do